project1/application.py

```diff
@@ -90,4 +90,22 @@ def authenticate():
 @app.route("/logout")
 def logout():
     session.clear()
-    return render_template("registration.html")
\ No newline at end of file
+    return render_template("registration.html")
+
+@app.route('/review', methods =['GET','POST'])
+def review():
+    if request.method == 'POST':
+        rating = request.form.get('review_tags')
+        review = request.form.get('review_value')
+        email = 'satish'
+
+        if (REVIEW.query.filter_by(email = email, isbn = "1234").first() == None) :
```

Comment:
is it for checking purpose you gave isbn value as 1234?

Reply:
yes maa'm, i gave static data of isbn as 1234

Reply:
Did you update the code?

Reply:
?

Reply:
yes ma'am i had updated the code